post-processing/streamlit_post_processing.py

```diff
@@ -68,6 +68,14 @@ def update_ui(post: PostProcessing, config: ConfigHandler, e: 'Exception | None'
         # config file uploader
         st.file_uploader("Upload Config", type="yaml", key="uploaded_config", on_change=update_config)
 
+        # set plot type
+        plot_type_options = ['generic', 'line']
+        plot_type_index = plot_type_options.index(config.plot_type) if config.plot_type else None
```

Comment:
```suggestion
        plot_type_index = plot_type_options.index(config.plot_type) if config.plot_type else 0
```
As plot type is required now, I'd make the fallback index 0 so that in the absence of information we default to a generic plot.

Reply:
I went for `None` so that the placeholder text is displayed and the user would then choose the plot type but I'm happy to default to generic if that's preferred?

Reply:
I think it is; the point of generic is that it should be able to display any other type of chart, so even if the user forgets to switch to a different plot type they'll get to see an output instead of a red screen.